inst/rmarkdown/resources/html/print_button.html

```diff
@@ -0,0 +1,7 @@
+<button id="print" style="left:36%;position:relative;background-color: #92bcbc94;">
+  <i class="glyphicon glyphicon-print" type="submit" onclick="window.PagedPolyfill.preview().then(() => {document.body.style.cssText='min-width: unset !important'; window.print();})" style= "padding-left: 5px; color: #cf581b;">
+  <b style = "color: #cf581b; padding-left: 5px;">Imprimer en A4</b>
```

Comment:
Il faut aussi déplacer le style inline dans une feuille de style

```suggestion
  <b style = "color: #cf581b; padding-left: 5px;">Imprimer</b>
```